assets/src/scss/layout/navbar/_search.scss

```diff
@@ -1,5 +1,5 @@
 .nav-search-form {
-  background-color: var(--body--background-color);
+  background: inherit;
```

Comment:
If you only desire to set the background color, background-color it's better than the shorthand background